README.md

```diff
@@ -44,6 +44,8 @@
 - **[A leaderboard API with unique ID]**
 - **[A submit button which sends user input to the API]**
 - **[A refresh button which returns the user scores]**
+- **[CSS styling]**
+- **[CSS media queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] To make your documentation looks more professional and descriptive, kindly ensure that  you update the `Getting started` section of your RADEME with all information needed by users in  order to run the app locally, such as `access` cloned directory, `install` dependencies, `run` the server,... To help you, I suggest you refer this **[Gist_file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**